tests/api/hfun.py

```diff
@@ -8,6 +8,7 @@
 
 from jigsawpy import jigsaw_msh_t
 import geopandas as gpd
+import matplotlib.pyplot as plt
```

Comment:
@SanderHulst can you please remove this since it's not being used for the test?

Reply:
I removed the unused requests and rasterio imports as well.